Gemfile

```diff
@@ -2,14 +2,18 @@ source "https://rubygems.org"
 git_source(:github){|repo| "https://github.com/#{repo}.git"}
 
 gem "bootsnap", ">= 1.4.2", require: false
+gem "config"
 gem "mysql2", ">= 0.4.4"
 gem "puma", "~> 4.1"
 gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+gem "rails-i18n"
+gem "sass-rails", ">= 6"
+gem "turbolinks", "~> 5"
```

Comment:
gem này có tác dụng gì?

Reply:
Turbolinks giúp cho việc di chuyển giữa các page trong trang web nhanh hơn.
Lý do là vì sau khi áp dụng turbolink thì khi các bạn click vào một link nào đó thuộc trang web của mình (links to the same domain), thì nó sẽ chỉ load lại phần body và merge header lại